src/packages/sdk/src/predefinedGeneratorResolvers.ts

```diff
@@ -1,14 +1,13 @@
+import Debug from '@prisma/debug'
 import chalk from 'chalk'
 import execa from 'execa'
 import fs from 'fs'
 import hasYarn from 'has-yarn'
 import path from 'path'
+import resolvePkg from 'resolve-pkg'
 import { logger } from '.'
 import { getCommandWithExecutor } from './getCommandWithExecutor'
-import { findUpAsync as findUp } from './utils/find'
-
-// hide require from bundlers
-const load = require
+const debugEnabled = Debug.enabled('generator')
```

Comment:
```suggestion
const debugEnabled = Debug.enabled('prisma:generator')
```
Better to prefix I think